packages/js/src/ml/HybridFilter.ts

```
@@ -0,0 +1,430 @@
+          };
+        }
+        return {
+          isToxic: false,
+          confidence: 0.85,
+          reason: 'No profanity detected (rule-based)',
+        };
+
+      case 'or':
+      default:
+        // Either method can flag
+        const isToxic = ruleDetected || mlDetected;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)